src/CattyTests/Bricks/BrickTests.swift

```diff
@@ -67,4 +67,34 @@ final class BrickTests: XCTestCase {
                        formulaManager.interpretDouble(brick.variableFormula, for: SpriteObject()),
                        "Invalid formula")
     }
+
+    func testMutableCopy() {
```

Comment:
Looks good! However, in the meantime this has been resolved in another (training) ticket. Sorry for the confusion, this should have been mentioned in your ticket. Please remove your two tests from this class. Additional, you could improve the existing tests in _BrickMutableCopyTests_ by adding the assertions for references as suggested below.